api/types/types.go

```diff
@@ -266,6 +266,8 @@ type ContainerJSONBase struct {
 	ExecIDs         []string
 	HostConfig      *runconfig.HostConfig
 	GraphDriver     GraphDriverData
+	SizeRw          int64 `json:",omitempty"`
```

Comment:
These two fields should be `*int64`, so they don't show up as `0` when the flag is not set.